torch/_dynamo/symbolic_convert.py

```diff
@@ -4176,14 +4203,18 @@ def log_graph_break(
             user_stack = torch._guards.TracingContext.extract_stack()
 
         try:
-            frame_loc = (user_stack[-1].filename, user_stack[-1].lineno)
```

Comment:
This code is actually pretty ancient - I wonder if we can just rely on information in the tx? i.e. treat the leaf tx the same as the others